rosdep/base.yaml

```
@@ -4624,6 +4624,125 @@ libpcl-all-dev:
   rhel: [pcl-devel]
   slackware: [pcl]
   ubuntu: [libpcl-dev]
+libpcl-common:
+  alpine: [pcl-libs]
```

could you add `arch: [pcl]`?

```suggestion
  arch: [pcl]
  alpine: [pcl-libs]
```